x/ccv/provider/client/cli/tx.go

```diff
@@ -65,3 +72,110 @@ func NewAssignConsumerKeyCmd() *cobra.Command {
 
 	return cmd
 }
+
+func NewSubmitConsumerMisbehaviourCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit-consumer-misbehaviour [misbehaviour]",
+		Short: "submit an IBC misbehaviour for a consumer chain",
+		Long: strings.TrimSpace(
```

Comment:
nit: is `strings.TrimSpace` needed here? My understanding is that it would just remove leading and trailing spaces but I do not see the underlying string to have any?

Reply:
Idnk the exact reason but It's kind of a convention used in the SDK cli.